dali/daliadmin/daadmin.cpp

```diff
@@ -275,13 +275,13 @@ void set(const char *path,const char *val)
     root->setProp(tail,val);
     conn->commit();
     root->getProp(tail,newv);
-    OUTLOG("Changed %s from '%s' to '%s'",path,oldv.str(),newv.str());
+    out.appendf("Changed %s from '%s' to '%s'",path,oldv.str(),newv.str());
```

Comment:
I'd prefer not to burn into the comment into the return string, let's have daliadmin output the message,
and have the lib setValue function just return the oldvalue.